packages/ibm-products/src/components/Datagrid/Datagrid/DatagridVirtualBody.tsx

```diff
@@ -84,13 +84,10 @@ const DatagridVirtualBody = (datagridState: DataGridState) => {
       const headWrapEl = document?.querySelector(
         `#${tableId} .${blockClass}__head-wrap`
       );
-      if (headWrapEl) {
+      if (headWrapEl && enableSpacerColumn) {
```

Comment:
I don't think you want to combine these conditions here. They should be separate like before, otherwise there is scrolling issues between the table body and table headers.

https://github.com/user-attachments/assets/e668afa4-bd95-4f56-b01c-ecde895a0b96

Reply:
You can also see there are multiple horizontal scroll bars